content/renderer/media/audio/mojo_audio_output_ipc.cc

```diff
@@ -260,16 +278,30 @@ void MojoAudioOutputIPC::Created(
 #if defined(CASTANETS)
 void MojoAudioOutputIPC::RequestTCPConnectCallback(
     base::UnsafeSharedMemoryRegion shared_memory_region,
-    int32_t port) {
-  // Create a TCP client socket.
-  mojo::PlatformHandle tcp_client_handle = mojo::CreateTCPClientHandle(port);
-  if (!tcp_client_handle.is_valid()) {
-    LOG(ERROR) << __func__ << " tcp_client_handle is not valid.";
-    return;
+    mojo::PlatformHandle server_handle,
+    uint16_t port) {
+  bool is_tcp_server = !port ? true : false;
```

Comment:
It seems that "!" mark isn't needed here.
bool is_tcp_server = !port ? true : false;

I guess you mean,
bool is_tcp_server = port ? true : false;

Reply:
If  "!" is removed, this will be changed to "bool is_tcp_server = port ? false : true;"

Reply:
For example, line 250, stream_->RequestTCPConnect(0, ...) is being called, 
and this call is in 
if (base::CommandLine::ForCurrentProcess()->HasSwitch(switches::kServerAddress)
block,
so I think, this call is for tcp client.
But with 0, 
bool is_tcp_server = !port ? true : false;
will set is_tcp_server = true.
So I'm confused a little.

Reply:
Hmm, this call was through mojo. So I mis-read this thing. Sorry for your inconvenience.